tests/common/dualtor/dual_tor_io.py

```diff
@@ -293,7 +294,10 @@ def generate_downstream_traffic(self, dst='server'):
             ip_ttl=ip_ttl,
             tcp_dport=TCP_DST_PORT
         )
-        tcp_tx_packet_orig = scapyall.Ether(str(tcp_tx_packet_orig))
+        if six.PY2:
```

Comment:
Can we wrap this logic into a helper function:
```suggestion
def convert_scapy_packet_to_bytes(packet):
    if six.PY2:
        return str(packet)
    else:
        return bytes(packet)
```
And maybe it is better to store it in a common module like `tests/common/scapy_utilities.py`

Reply:
Will wrap this logic into a helper function.
Will commit later after I finish running all test cases under dualtor_io.